### PR TITLE
Create meta cache on task creation

### PR DESCRIPTION
**Pull request by alexanderGerbik**
https://p.daedalean.ai/T6188

### CODE REVIEW

File: cvat/apps/engine/task.py [hunk 1]
```diff
@@ -340,6 +340,11 @@ def _create_thread(tid, data, options):
     else:
         segments = []
     _save_task_to_db(db_task, segments)
+
+    job.meta['status'] = 'Image meta cache is being created'
+    job.save_meta()
+    make_image_meta_cache(db_task)
```
Comment (gyakushev) [inline cvat/apps/engine/task.py:346]: I don't see this function defined anywhere
did you add all files to this pull request?

Comment (alexanderGerbik) [reply cvat/apps/engine/task.py:346]: this function was written by cvat authors and defined there:
https://github.com/daedaleanai/cvat/blob/62f942f4253120b00e4cdf78156a2dfd349f35aa/cvat/apps/engine/task.py#L52-L81

Comment (gyakushev) [reply cvat/apps/engine/task.py:346]: Thanks! GitHub search failed me here for some reason.

**Review by gyakushev — APPROVED** [review]

File: cvat/apps/engine/task.py (hunk 1 above)
Comment (alexanderGerbik) [reply cvat/apps/engine/task.py:346]: It turns out that forked repositories are not searchable, but search in the parent repository works.

**Pull request merged**